README.md

```diff
@@ -11,66 +11,66 @@ Search [Crystal Shards](https://crystalshards.xyz) or follow announcements [Crys
 
 Contributions are welcome. Please take a quick look at the [contribution guidelines](https://github.com/veelenga/awesome-crystal/blob/master/.github/CONTRIBUTING.md) first.
 
-* [Awesome Crystal](#awesome-crystal)
-  * [Algorithms and Data structures](#algorithms-and-data-structures)
```

Comment:
Pls revert these changes back. If you want to suggest changing it please open a separate pr